### PR TITLE
Additional details on Chrome storage availability

### PR DESCRIPTION
**Pull request by petele**
Adds additional details about the amount of storage quota available to web sites in Chrome.

### CODE REVIEW

File: src/site/content/en/blog/storage-for-the-web/index.md [hunk 1]
```diff
@@ -96,9 +94,14 @@ device.
 * Chrome allows the browser to use up to 80% of total disk space. An origin can
   use up to 60% of the total disk space. You can use the [StorageManager
   API](#check) to determine the maximum quota available. Other Chromium-based
-  browsers may allow the browser to use more storage. See
-  [PR #3896](https://github.com/GoogleChrome/web.dev/pull/3896) for details about
-  Chrome's implementation.
+  browsers may allow the browser to use more storage.
```
Comment (jpmedley) [inline src/site/content/en/blog/storage-for-the-web/index.md:97]: Browsers may allow the browser? If this is what you meant, I'm not sure I understand.

File: src/site/content/en/blog/storage-for-the-web/index.md [hunk 2]
```diff
@@ -96,9 +94,14 @@ device.
...
+  * In Incognito Mode, Chrome reduces the amount of storage an origin can use
```
Comment (jpmedley) [inline src/site/content/en/blog/storage-for-the-web/index.md:98]:
```suggestion
  * In incognito mode, Chrome reduces the amount of storage an origin can use
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: src/site/content/en/blog/storage-for-the-web/index.md (hunk 1 above)
Comment (petele) [reply src/site/content/en/blog/storage-for-the-web/index.md:97]: `Other Chromium-based browsers may allow the browser to use more storage.`

When read as the whole sentence, I think it makes sense, but I'll update it.

**Comment by petele** [comment]
@jpmedley - I've updated based on your feedback.

**Pull request merged**